lib/logger.js

```diff
@@ -0,0 +1,11 @@
+const cds = require("@sap/cds");
+
+const Logger = cds.log("ord-plugin", {
+    level: cds.env.DEBUG || process.env.DEBUG
```

Comment:
@zongqichen : log level depends on DEBUG variable that can be set in two ways: as environment variable or in .npmrc.json document.
If it's value is truthy, the level is DEBUG, which means that all the messages with level DEBUG and higher will be shown in console; 
If it's value is not truthy, the default level is WARN, meaning that only logs with severity WARN and higher will be shown in the console.

Reply:
@zongqichen : This way we instantiate a single instance of logger and set it's level and name(label) **centrally**.

Reply:
@zongqichen : 
```
const logger1 = cds.log("some name");
const logger2 = cds.log("some name");

console.log(`The statement that logger1 is equal to logger2 is ${logger1 === logger2}.`); 
// result: "The statement that logger1 is equal to logger2 is true."
```

Reply:
Thanks, I like the logger.js file. And I'm surprised that `logger1 === logger2`, need to refresh my knowledge.

Reply:
> I'm surprised that `logger1 === logger2`, need to refresh my knowledge.
Yes, this is the link: https://pages.github.tools.sap/cap/docs/node.js/cds-log#logger-id
```text
The loggers constructed by cds.log() are cached internally, and the same instances are returned on subsequent
 invocations of cds.log() with the same id. This allows to use and share the same logger in different modules.
```